Initial Commit (#4)
armorblox/Azure-Sentinel: Solutions/Armorblox/Playbooks/README.md

# Needs-Review-Incident-Email-Notification
**Author:** Armorblox

> This playbook will send an Email notification when a new incident is opened in Azure Sentinel.

Reviewer: ```suggestion
This playbook will send an email notification when a new incident is created in Azure Sentinel.
```

## Pre-requisites

> An O365 account to be used to send email notification (The user account will be used in O365 connector (Send an email).)

Reviewer: What is the `(Send an email)` bit?

Author: It represents the step 2 in the playbook. o365 is the brand and the feature used from the brand is `send an email`

Author: Should I remove it `(Send an email)` ?

Reviewer: That's fine. I think you can change this line to `An O365 account to be used to send email notification. The user account will be used in O365 connector (Send an email).`

Author: Done

## Parameters
Notification Email - The receiver's mail address.

## Deployment                                                                                         
[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Farmorblox%2FAzure-Sentinel%2Fxoriant%2FSolutions%2FArmorblox%2FPlaybooks%2Fazuredeploy.json)
[![Deploy to Azure Gov](https://aka.ms/deploytoazuregovbutton)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Farmorblox%2FAzure-Sentinel%2xoriant%2FSolutions%2FArmorblox%2FPlaybooks%2Fazuredeploy.json)
 

## Post-deployment

### Configure connections

> Once this Playbooks template is deployed, you will need to go into the Logic App, edit it and click on each of the steps that require an authenticated connection to your tenant and complete the connection process. These steps will have an exclamation point showing that the connection needs to be completed. Make sure to also open the "For each" step which also contains a step that requires an authenticated connection.

Reviewer: ```suggestion
Once this Playbooks template is deployed, you will need to go into the Logic App, edit it and click on each of the steps that require an authenticated connection to your tenant and complete the connection process. These steps will have an exclamation mark showing that the connection needs to be completed. Make sure to also open the "For each" step which also contains a step that requires an authenticated connection.
```

> Note:  Email sent with this playbook will be from user that creates connection!

Reviewer: ```suggestion
Note:  Emails sent with this playbook will be from the user that creates the connection.
```

### Attach the playbook
After deployment, attach this playbook to the `Armorblox Needs Review` analytic rule so it runs when the alert is created.</br>

> Note: Playbook is disabled by default. Please enable it before assigning to the Automation rule!

Reviewer: ```suggestion
Note: Playbook is disabled by default. Please enable it before assigning to the Automation rule.
```
